src/calendar/view/eventeditor/AttendeeListEditor.ts

```diff
@@ -136,6 +133,11 @@ export class AttendeeListEditor implements Component<AttendeeListEditorAttrs> {
 			attendeeRenderers.push(() => renderGuest(guest, attrs))
 		}
 
+		const ownGuest = whoModel.ownGuest
```

Comment:
I think this chunk needs a comment on it. Is own guest not in the list of guests? why is this needed?

Reply:
whoModel.guests actually specifies that on the getter, but I added another comment.